clio-server/src/main/resources/reference.conf

```diff
@@ -15,6 +15,7 @@ clio.server {
   }
 
   elasticsearch {
+    replicate-indices: true
```

Comment:
I don't want anyone to ever have to touch this value, or even fiddle with a `number_of_replicas`. It only exists so that the unit tests can run on as single node. The disabling of the index replication allows the single ES node to return a "green" status.

If we take out the various checks for "green", then this will not be needed.